src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import classNames from 'classnames';
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+type Props = {
+  people: Person[];
+  error: boolean;
+};
+
+export const PeopleTable: React.FC<Props> = ({ people, error }) => {
+  const { slug: selectedSlug } = useParams();
+
+  return (
+    <>
+      {error && (
+        <p data-cy="peopleLoadingError" className="has-text-danger">
+          Something went wrong
+        </p>
+      )}
+
+      {people.length < 0 ? (
```

Comment:
Replace 'people.length < 0' with 'people.length === 0' to correctly check for an empty array of people.